Jenkinsfile

```diff
@@ -0,0 +1,60 @@
+#!/usr/bin/groovy
+
+@Library(['github.com/indigo-dc/jenkins-pipeline-library@release/2.1.0']) _
+
+def projectConfig
+
+pipeline {
+    agent any
+
+    environment {
+        dockerhub_credentials = "o3as-dockerhub-vykozlov"
+        cicd_backend_version = "1.0.1"
+        dockerhub_cicd_backend = "eoscperf/cicd-images:backend-${cicd_backend_version}"
+    }
+
+    stages {
+        stage('Build Docker image with tools for CI/CD') {
```

Comment:
I see this is the stage to create the container on dockerhub.
Is this needed? Can't the pipeline just build the container? 

Well maybe it is because it would speed up things as the tools would be there already, but would not be the same as the docker steps are cached? Maybe not in the jenkins server.....

Reply:
I think `docker-compose up --build` used by JePL triggers rebuild of the whole docker image. If the corresponding Dockerfile starts with "FROM python:XXX", it goes through all the steps. Having "static" eoscperf/cicd-images:backend-x.y.z and starting "FROM eoscperf/cicd-images:backend-x.y.z" allows to avoid all steps of installing postgres etc every time.

The stage "Build Docker image with tools for CI/CD" is executed *only* if Jenkinsfile OR Dockerfile.cicd changed.